src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1581,11 +1581,14 @@
   <Node Name="BoundWildcardPattern" Base="BoundPattern">
   </Node>
 
-  <!-- The node is transformed into BoundLocal after inference -->
-  <Node Name="OutVarLocalPendingInference" Base="BoundExpression">
+  <!-- The node is transformed into BoundLocal or BoundFieldAccess after inference -->
+  <Node Name="OutVariablePendingInference" Base="BoundExpression">
```

Comment:
This is in conflict with the bound node design guidelines. If there is a declaration at this point, there should be a bound node that describes the declaration of that symbol. The transformation described here makes the bound node indistinguishable from a reference to a local declared elsewhere, and belongs more properly in the lowering phase.